src/Controls/src/Core/Brush/Brush.cs

```diff
@@ -116,6 +116,30 @@ public static bool IsNullOrEmpty(Brush brush)
 			return brush == null || brush.IsEmpty;
 		}
 
+		// TODO: Make this method public in .NET 11
+		internal static bool HasTransparency(Brush background)
```

Comment:
Could include some unit tests? Unit tests covering SolidColorBrush with alpha, GradientBrush with opaque and mixed stops, and null/empty brushes to lock behavior